storagegroup/storagegroup.go

```diff
@@ -234,8 +234,8 @@ func (sg *StorageGroup) SetMembers(members []oid.ID) {
 // Marshal marshals StorageGroup into a protobuf binary form.
 //
 // See also Unmarshal.
-func (sg StorageGroup) Marshal() ([]byte, error) {
-	return (*storagegroup.StorageGroup)(&sg).StableMarshal(nil), nil
+func (sg StorageGroup) Marshal() []byte {
```

Comment:
i would drop all at once. this was is also ok to me

Reply:
gone after rebase